config/notifiers.go

```diff
@@ -142,6 +143,7 @@ type EmailConfig struct {
 	AuthIdentity string            `yaml:"auth_identity,omitempty" json:"auth_identity,omitempty"`
 	Headers      map[string]string `yaml:"headers,omitempty" json:"headers,omitempty"`
 	HTML         string            `yaml:"html,omitempty" json:"html,omitempty"`
+	TEXT         string            `yaml:"text,omitempty" json:"text,omitempty"`
```

Comment:
Hi @fabxc 

I am not sure I understand your comment. I am not a Go dev (yet) so I need more details about what you are requesting as a change.

Also regarding the package for multi-part. The original code of email notification was constructed manually from smtp handshake to email construction. I am not at the level to rework this entire section of code. The smtp protocole is old dated and is not likely to change any time soon. may I recommend adding //TODO refactor to use standard multipart library https://godoc.org/mime/multipart?